src/main/webapp/authentication-app/src/firebase.js

```diff
@@ -0,0 +1,19 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+const firebaseConfig = {
+    apiKey: "AIzaSyAjse2YjCJHUUe5w1i7I9jkjH8cqqRcAIU",
```

Comment:
Was the consensus on Firebase api keys that they should be embedded in client-side code? Can we not handle this server-side and protect the API key now?